README.md

```diff
@@ -0,0 +1,55 @@
+# Inbox - The simplest Aptos backend
+
+Inbox is a simple backend server that automatically creates a REST API for your
+contract using your events.
+
+## Dependencies
+
+The only dependency you need is Docker.
+
+## Setting up
+
+### Step 0: Cloning
+
+First, clone this project: `git clone https://github.com/econia-labs/inbox`.
+
+Then, clone git submodules: `git submodule update --init --recursive`.
+
+### Step 1: Configuration
+
+Copy `example.env` to `.env` and update the required fields. Field
+documentation can be found in the file.
+
+### Step 2: Run
+
+Simply start Inbox by running `docker compose -p inbox -f compose.yaml up -d`.
+
+## Using
+
```

Comment:
Maybe a note about the default database connection string?

`postgres://inbox:inbox@postgres:5432/inbox`